kolibri/core/assets/src/core-actions.js

```diff
@@ -350,8 +350,10 @@ function initContentSession(store, coreApp, channelId, contentId, contentKind) {
 function _setChannelState(store, coreApp, currentChannelId, channelList) {
   store.dispatch('SET_CORE_CHANNEL_LIST', channelList);
   store.dispatch('SET_CORE_CURRENT_CHANNEL', currentChannelId);
-  coreApp.resources.ContentNodeResource.setChannel(currentChannelId);
-  cookiejs.set('currentChannelId', currentChannelId);
+  if (currentChannelId !== null) {
+    coreApp.resources.ContentNodeResource.setChannel(currentChannelId);
+    cookiejs.set('currentChannelId', currentChannelId);
+  }
```

Comment:
If

```javascript
currentChannelId === null
```

we might want to actually

```javascript
cookiejs.remove('currentChannelId');
```

(If the user previously had a cookie set, we probably don't want that to be used in the future).